lib/hexpm/web/templates/docs/public_keys.html.md

```diff
@@ -34,4 +34,4 @@ No public keys have been revoked yet.
 
 #### Mix
 
-Run `mix help hex.public_keys` in your terminal for instructions or go to [`hex.public_keys`](/docs/tasks#hex_public_keys).
+Run `mix help hex.user` in your terminal for instructions or go to [`hex.user`](/docs/tasks#hex_user).
```

Comment:
Nice catch but we should point to the `hex.repo` tasks since that's where you set up repository public keys. Thanks!